ci/deps/azure-38-locale.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - html5lib
   - ipython
   - jinja2
+  - jedi<0.18.0
```

Comment:
tiny thing but alphabetically jedi is before jinja

Reply:
good catch. I think the intention is to revert shortly, so this is OK for now. (to get CI green)